### PR TITLE
Fontionality popin

### PR DESCRIPTION
**Pull request by almerion**

### CODE REVIEW

File: src/css/home-page.css [hunk 1]
```diff
@@ -166,4 +166,24 @@ article{
 .logo{
     width: 70px;
     vertical-align: text-top;
+}
+
+
+#popInInfoDiv {
```
Comment (NaulaN) [inline src/css/home-page.css:172]:
```suggestion
#pop-in-info-div {
```
or, we needs to change all attributes from `xx-xx` to `xxXx`

File: src/css/home-page.css [hunk 2]
```diff
@@ -166,4 +166,24 @@ article{
...
+    position: absolute;
+    top : 60px;
+    right: 0;
+    width: 10%;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+}
+.popInElement {
```
Comment (NaulaN) [inline src/css/home-page.css:182]: Same here

File: src/index.php [hunk 3]
```diff
@@ -1,31 +1,31 @@
...
+                $module = new ModHome();
+                break;
+            default :
+                break;
+        }
+        $content = $module->getDisplay();
+        ?>
 
+    <!DOCTYPE html>
+    <html lang="fr">
```
Comment (NaulaN) [inline src/index.php:28]: Why you change position of `<?php ?>` tags ?

File: src/index.php [hunk 4]
```diff
@@ -38,4 +38,5 @@
     <body>
         <?= $content ?>
     </body>
+    <script src="./main.js"></script>
```
Comment (NaulaN) [inline src/index.php:41]: Wrong path, you need to move on `./js/main.js`
```suggestion
    <script src="./js/main.js"></script>
```

File: src/main.js [hunk 5]
```diff
@@ -0,0 +1,20 @@
+function popin(text,isPermanent = true){
+    let div = document.getElementById("popInInfoDiv");
```
Comment (NaulaN) [inline src/main.js:2]: This variable not going to change so let's go change to `const`
```suggestion
    const div = document.getElementById("popInInfoDiv");
```

File: src/js/css_loader.js [hunk 6]
```diff
@@ -27,8 +26,10 @@ window.onload = function()
         const nameVar = urlVars[i].split('=')[0];
         const valueVar = urlVars[i].split('=')[1];
 
-        if (nameVar === "status") {
-            createLinkCSS(pathsCSS[valueVar]);
-        } else createLinkCSS(pathsCSS["home"]); // TODO Temporary
+        if (nameVar === 'module' && valueVar === 'user') {
+            createLinkCSS(pathsCSS['loginRegister']);
+        } else if (nameVar === 'module' && valueVar === 'home') {
```
Comment (NaulaN) [inline src/js/css_loader.js:31]: You repeat two times `nameVar === 'module'`

File: src/main.js [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+    let elem = document.createElement("div");
```
Comment (NaulaN) [inline src/main.js:3]:
```suggestion
    const elem = document.createElement("div");
```

File: src/main.js [hunk 8]
```diff
@@ -0,0 +1,20 @@
...
+    let paragraph = document.createElement("p");
```
Comment (NaulaN) [inline src/main.js:4]:
```suggestion
    const paragraph = document.createElement("p");
```

File: src/main.js [hunk 9]
```diff
@@ -0,0 +1,20 @@
...
+
+
+    paragraph.textContent = text;
+    paragraph.style.textAlign = 'center';
+
+    elem.appendChild(paragraph);
+    div.insertBefore(elem,div.firstChild);
+    elem.classList.add("popInElement");
+
+    if (isPermanent === false) {
```
Comment (NaulaN) [inline src/main.js:14]:
```suggestion
    if (!isPermanent) {
```

File: src/modules/mod_user/vue_user.php [hunk 10]
```diff
@@ -59,6 +60,7 @@ public function registerForm()
             <button id="back-button" class="buttons-form" type="button">RETOUR</button>
         </div>
     </form>
+    <div id="popInInfoDiv" class="test"></div>
```
Comment (NaulaN) [inline src/modules/mod_user/vue_user.php:63]: bruh, `class="test"`

File: src/modules/mod_user/vue_user.php [hunk 11]
```diff
@@ -75,10 +77,13 @@ public function loginAlreadyTaken()
...
-    {
-        # code...
-    }
-
+    { ?>
+        <script type="text/javascript" defer>
+            window.addEventListener('load', function () {
+                popin('Les informations entrées sont érronées, veuillez réessayer');
+            })
+        </script>
```
Comment (NaulaN) [inline src/modules/mod_user/vue_user.php:85]: You can create a `.js` file and put your script on it ?

**Review by NaulaN — CHANGES_REQUESTED** [review]

File: src/index.php (hunk 3 above)
Comment (almerion) [reply src/index.php:28]: image
:moyai:

Comment (NaulaN) [reply src/index.php:28]: mb

File: src/modules/mod_user/vue_user.php (hunk 11 above)
Comment (almerion) [reply src/modules/mod_user/vue_user.php:85]: done

File: src/modules/mod_user/vue_user.php (hunk 10 above)
Comment (almerion) [reply src/modules/mod_user/vue_user.php:63]: exact, bruh, i removed it

File: src/index.php (hunk 4 above)
Comment (almerion) [reply src/index.php:41]: fixed

File: src/css/home-page.css (hunk 2 above)
Comment (almerion) [reply src/css/home-page.css:182]: adopted the kebab-case

File: src/css/home-page.css (hunk 1 above)
Comment (almerion) [reply src/css/home-page.css:172]: Adopted the kebab-case

**Review by NaulaN — CHANGES_REQUESTED** [review]

File: src/js/css_loader.js [hunk 12]
```diff
@@ -26,10 +26,12 @@ window.onload = function()
         const nameVar = urlVars[i].split('=')[0];
         const valueVar = urlVars[i].split('=')[1];
 
-        if (nameVar === 'module' && valueVar === 'user') {
-            createLinkCSS(pathsCSS['loginRegister']);
-        } else if (nameVar === 'module' && valueVar === 'home') {
-            createLinkCSS(pathsCSS["home"]); // TODO Temporary
+        if (nameVar === 'module') {
+            if ( valueVar == "user") {
```
[the remark this answers is not in the document]
Comment (almerion) [reply src/js/css_loader.js:30]: it's because of Joe

**Review by NaulaN — APPROVED** [review]

File: src/js/css_loader.js (hunk 12 above)
[the remark this answers is not in the document]
Comment (almerion) [reply src/js/css_loader.js]: Joe mama

**Pull request merged**